styles.css

```diff
@@ -0,0 +1,555 @@
+  font-size: 30px;
+  font-weight: 900;
+}
+
+.italic {
+  font-style: italic;
+}
+
+.fontSize12 {
+  font-size: 12px;
+}
+
+.fontsize14 {
+  font-size: 14px;
+}
+
+.fontSize18 {
+  font-size: 18px;
+}
+
+.fontSize20 {
+  font-size: 20px;
+}
+
+.fontSize24 {
+  font-size: 24px;
+}
+
+.fontSize36 {
+  font-size: 36px;
+}
+
+.fontWeight700 {
+  font-weight: 700;
+}
+
+.mainProgram {
+  background-color: #272a31;
+}
+
+.lineHeight15 {
+  line-height: 1.5;
+}
+
+.mLeft20 {
+  margin-left: 20px;
+}
+
+.mRight20 {
+  margin-right: 20px;
+}
+
+.hamburger {
+  padding-left: 32px;
+  padding-top: 24px;
+  padding-bottom: 24px;
+}
+
+.welcomeText {
+  margin-top: 40px;
+}
+
+.subText {
+  margin: 20px 10px;
+  padding: 10px;
+  border: 3px solid white;
+}
+
+.location {
+  margin-top: 10px;
+  margin-bottom: 100px;
+}
+
+.program {
+  padding-top: 36px;
+  padding-bottom: 48px;
+  text-align: center;
+}
+
+.programStyle {
+  margin: 16px;
+  padding: 16px;
+  background-color: #3c3f46;
+  gap: 10px;
+  align-items: center;
+}
+
+.joinButton {
+  background-color: #ec5242;
+  padding-left: 20%;
+  padding-right: 20%;
+  padding-top: 24px;
+  padding-bottom: 24px;
+  margin-top: 24px;
+}
+
+.container1 {
+  padding-bottom: 48px;
+}
+
+.playerList {
+  padding-top: 36px;
+  padding-bottom: 60px;
+  text-align: center;
+}
+
+.playerImg {
+  float: left;
+  margin: 0 16px 12px;
+}
+
+.playerAchivement {
+  margin: 8px 0;
+}
+
+.width20 {
+  width: 45px;
+}
+
+.red {
+  color: red;
+}
+
+.moreBtn {
+  padding-top: 16px;
+  padding-bottom: 16px;
+  margin: 24px 16px;
+  flex: 1;
+  background-color: rgb(248, 248, 248);
+}
+
+.textCenter {
+  text-align: center;
+}
+
+.welcomeTextAbout {
+  margin-top: 40px;
+  margin-bottom: 12px;
+}
+
+.aboutDescription {
+  background-color: white;
+  margin: 36px 16px 16px;
+  text-align: center;
+  padding: 18px 12px;
+}
+
+.aboutLogo {
+  margin-top: 24px;
+  background-color: white;
+}
+
+.aboutLogoTitle {
+  padding-top: 60px;
+  padding-bottom: 24px;
+}
+
+.aboutLogoTitleCompetition {
+  margin: 0 60px;
+}
+
+.logoImg {
+  border: solid black 2px;
+  margin: 16px 16px 0;
+}
+
+.buc {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(249, 52, 52, 0.5),
+      rgba(249, 52, 52, 0.5)
+    ),
+    url(./img/img1.jpg);
+  background-repeat: no-repeat;
+  width: 380px;
+  height: 210px;
+  padding: 0;
+  border: 0;
+  margin: 0;
+}
+
+.bra {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(249, 52, 52, 0.5),
+      rgba(249, 52, 52, 0.5)
+    ),
+    url(./img/img2.jpg);
+  background-repeat: no-repeat;
+  width: 380px;
+  height: 210px;
+  padding: 0;
+  border: 0;
+  margin: 0;
+}
+
+.cityName {
+  font-size: 34px;
+  font-weight: 900;
+}
+
+.cityNameText {
+  font-size: 12px;
+  font-weight: 900;
+}
+
+.backgroundGray {
+  background-color: #3c3f46;
+}
+
+.asa {
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
```

Comment:
- [ ] Kindly layout your `images` side by side as seen in the Original design 🙏 

Original Behance Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190199134-6b5b2511-df27-4006-a6fa-8f8dc490071e.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190198983-02e6f19b-0fc9-4562-93f8-b97992b0a488.png)